tools/c7n_gcp/c7n_gcp/resources/resourcemanager.py

```diff
@@ -200,11 +259,16 @@ def load_hierarchy(self, resources):
 
         for r in resources:
             client = self.get_client(session, self.manager.resource_type)
-            ancestors = client.execute_command(
-                'getAncestry', {'projectId': r['projectId']}).get('ancestor')
-            parents[r['projectId']] = [
-                a['resourceId']['id'] for a in ancestors
-                if a['resourceId']['type'] == 'folder']
+            if 'folders' in r:
```

Comment:
asset inventory auto populates hierarchy for folders, which obviates need to fetch